lardataobj/Simulation/classes_def.xml

```diff
@@ -33,6 +33,11 @@
  </class>
  <class name="std::vector< sim::SimEnergyDeposit>"/>
  <class name="art::Wrapper< std::vector< sim::SimEnergyDeposit>>"/>
+ <class name="sim::SimEnergyDepositLite" ClassVersion="13" >
+  <version ClassVersion="13" checksum="3186052868"/>
+ </class>
```

Comment:
Because this is the first version to be included in `lardataobj`, can you reset the `ClassVersion`  number to 10?

Reply:
Done!